.github/workflows/python-tests.yml

```
@@ -0,0 +1,72 @@
+name: Python Tests
+
+# 1. Run on every push 
+on:
+  push:
+    branches:
+      - '**'
+  pull_request:
+    branches:
+      - main
```

minor nitpick, but the `pull_request` config isn't necessary if we're running CI on every push
```suggestion
on:
  - push
```